deps-tasks.yml

```diff
@@ -130,6 +133,7 @@ tasks:
       DEST: "{{.DEST_DIR}}/antlr-4.13.1-complete.jar"
     sources:
       - "{{.G_DEP_DOWNLOAD_SCRIPT}}"
+      - "{{.ROOT_DIR}}/Taskfile.yml"
```

Comment:
Why do we need to depend on the upper level taskfile?

Reply:
The task is depending on `:init` which is in the main Taskfile. That said:
* I don't think we need `:init` in these tasks.
* We are depending on the utility tasks so we should be depending on that Taskfile.

Reply:
logically it makes sense. It sounds weird that changing anything in the main taskfile will force the download dependency task to rerun. but i guess it's ok since the checksum will prevent the task from actually rerunning.

Reply:
Added dependency on utility taskfile